engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/jni/IValueUtils.java

```diff
@@ -66,7 +66,7 @@ public static NDList forward(PtSymbolBlock block, NDList inputs, boolean isTrain
      * @param inputs the input {@link IValue}
      * @return the result {@link IValue}
      */
-    public static IValue forward(PtSymbolBlock block, IValue... inputs) {
+    public static IValue forward(PtSymbolBlock block, IValue[] inputs) {
```

Comment:
I don't understand this change. You should be able to pass in an array of data to the `Ivalue... inputs` anyway